src/utils/DataSourceHelper/DataSourceHelper.ts

```diff
@@ -0,0 +1,112 @@
+import "./DataSourceHelper.scss";
+
+interface ConstraintStore {
+    [widgetId: string]: string | HybridConstraint;
+}
+
+interface Version {
+    major: number; minor: number; path: number;
+}
+
+interface HybridConstraint {
+    attribute: string;
+    operator: string;
+    value: string;
+    path?: string;
+}
+
+interface ListView extends mxui.widget._WidgetBase {
+    _datasource: {
+        _constraints: HybridConstraint[] | string;
+    };
+    datasource: {
+        type: "microflow" | "entityPath" | "database" | "xpath";
+    };
+    update: (obj: mendix.lib.MxObject | null, callback?: () => void) => void;
+    _entity: string;
+}
+
+export class DataSourceHelper {
+    static VERSION: Version = { major: 1, minor: 0, path: 0 };
+    version: Version = DataSourceHelper.VERSION;
```

Comment:
Confused... why do you need both of these?

Reply:
As explained previously. At the time we needed to know the version of dataSourceHelper's instance. so it could easily be accessible via a public version.